tool/tsh/common/device.go

```diff
@@ -75,16 +80,24 @@ func newDeviceCommand(app *kingpin.Application) *deviceCommand {
 type deviceEnrollCommand struct {
 	*kingpin.CmdClause
 
-	token string
+	currentDevice bool
+	token         string
 }
 
 func (c *deviceEnrollCommand) run(cf *CLIConf) error {
+	if c.token == "" && !c.currentDevice {
```

Comment:
There's a question here of whether we let `c.currentDevice` be the default behavior, or ask for it explicitly. I chose the latter because I expect the number of users that can use it to be fairly small (only people that can do all kinds of device actions will get through it).

If the focus is day1 above else, then we can make it the default.

Reply:
I like making it explicit, so long as our day-1 getting started guide says to use `--current-device`.